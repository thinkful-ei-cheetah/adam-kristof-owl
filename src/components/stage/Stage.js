import React from 'react';
import './Stage.css';

export default function Stage (props){
  console.log(props);
  const inStage = props.inStage.map(staged =>{
    return (
      <li key={staged.id}>
        <div>{staged.name}</div>
        <img src={staged.avatar}/>
      </li>
    )
  });

  return(
    <div className="vid">
      <ul>
        {inStage}
      </ul>
    </div>
  )

}