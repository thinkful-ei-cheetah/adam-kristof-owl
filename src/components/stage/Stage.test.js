import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage'
import renderer from 'react-test-renderer'  


describe('Stage Componenet working', ()=>{
  it('Renders to the page without crashing', ()=>{
    const div = document.createElement('div')
    ReactDOM.render(<Stage  inStage={[]}/>, div)
    ReactDOM.unmountComponentAtNode(div)
  });
  
  it('Renders as it did the first time!', ()=>{
    const tree = renderer
    .create(<Stage inStage={[]}/>)
    .toJSON()
    expect(tree).toMatchSnapshot();
  });
});
