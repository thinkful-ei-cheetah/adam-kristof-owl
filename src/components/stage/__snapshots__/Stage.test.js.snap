// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage Componenet working Renders as it did the first time! 1`] = `
<div
  className="vid"
>
  <ul />
</div>
`;
